main.go

```diff
@@ -272,27 +257,27 @@ func onResolvedGroup(data template.Data, updatableIncident Incident) error {
 func alertGroupToIncident(data template.Data) (Incident, error) {
 
 	incident := Incident{
-		"assignment_group":                    config.DefaultIncident.AssignmentGroup,
-		"category":                            config.DefaultIncident.Category,
-		"contact_type":                        config.DefaultIncident.ContactType,
 		"caller_id":                           config.ServiceNow.UserName,
-		"cmdb_ci":                             config.DefaultIncident.CmdbCI,
-		"comments":                            config.DefaultIncident.Comments,
-		"company":                             config.DefaultIncident.Company,
-		"description":                         config.DefaultIncident.Description,
-		"impact":                              config.DefaultIncident.Impact,
-		"short_description":                   config.DefaultIncident.ShortDescription,
 		config.Workflow.IncidentGroupKeyField: getGroupKey(data),
-		"subcategory":                         config.DefaultIncident.SubCategory,
-		"urgency":                             config.DefaultIncident.Urgency,
 	}
 
-	applyIncidentTemplate(incident, data)
-	validateIncident(incident)
+	for k, v := range config.DefaultIncident {
+		incident[k] = v
+	}
 
+	errs := applyIncidentTemplate(incident, data)
+	logErrors(errs)
+	errs = validateIncident(incident)
+	logErrors(errs)
 	return incident, nil
 }
 
+func logErrors(errs []error) {
+	for err := range errs {
```

Comment:
This is not working, should be something like:
```suggestion
	for _, err := range errs {
```

Reply:
This isn't a comment on functionality but more like a personal opinion: I think we should return a single error in all of these cases and use the idiomatic GO error handling pattern instead. I think it's more readable that way.